modules/opentracing/src/main/scala/OTSpan.scala

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2019 by Rob Norris
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package natchez
+package opentracing
+
+import cats.effect.{Resource, Sync}
+import cats.implicits._
+import io.opentracing.propagation.{Format, TextMapAdapter}
+import io.{opentracing => ot}
+import natchez.opentracing.OTTracer.makeSpan
+
+import scala.jdk.CollectionConverters._
+
+private[opentracing] final case class OTSpan[F[_] : Sync](tracer: ot.Tracer,
+                                                          span: ot.Span) extends Span[F] {
+  import TraceValue._
+
+  def kernel: F[Kernel] =
```

Comment:
Defs without parameters can be vals. As a bonus, we get fewer thunk allocations in `delay` (reusing the same thunk for the lifetime of this OTSpan instance)